### DISCUSSION TITLE
Use SQLite by default for Cipi dashboad

### DISCUSSION THREAD

**Discussion by arwinvdv** [andreapollastri/cipi#436] [Ideas]
Hi,

If the MySQL service is down, you can't login into the Cipi dashboard. Maybe use SQLite so you can restart the service in the dashboard?

**Comment by andreapollastri** [comment]
Hello guys, it the last 3 years the world has been changed with AI and with new deployments needs so a new "Laravel Control Panel" is not the real solution for today's world. I have created a new stable version of Cipi full of features and based on a powerful and light CLI. If you would like to try it, here you can find all informations: https://cipi.sh/
